source/usage/other/craftscripts.rst

```diff
@@ -47,54 +45,128 @@ Scripts have the following three variables in their global namespace:
 * ``player`` is the player who invoked the script, an instance of `Player <https://github.com/EngineHub/WorldEdit/blob/master/worldedit-core/src/main/java/com/sk89q/worldedit/entity/Player.java>`_.
 * ``argv`` is a Java array strings, which are the arguments used upon invoking the scripts
 
-Working with blocks
+In addition to these globals, if you wish to import any WorldEdit or Java packages or classes you must use the function `importPackage` or `importClass`, respectively:
+
+.. topic:: Example: Importing packages and classes
+
+    ::
+
+        // Equivalent to `import com.sk89q.worldedit.world.*`
+        importPackage(Packages.com.sk89q.worldedit.world);
+        // Equivalent to `import com.sk89q.worldedit.world.block.*`
+        importPackage(Packages.com.sk89q.worldedit.world.block);
+        // Equivalent to `import java.utils.*`
+        importPackage(java.util)
+
+        // Equivalent to`import com.sk89q.worldedit.math.BlockVector3`
+        importClass(Packages.com.sk89q.worldedit.math.BlockVector3);
+        // Equivalent to `import java.utils.ArrayList`
+        importClass(java.util.ArrayList)
+
+Warning: unlike Java, Rhino JavaScript does not implicitly import ``java.lang.*``, and doing so explicitly is not permitted.
```

Comment:
What does this mean?
```js
importPackage(Packages.java.lang);

player.print("yo! " + System.getenv())
```
proves this false right away. Importing `java.lang` is fine.

Reply:
Fixed in https://github.com/EngineHub/WorldEditDocs/pull/13/commits/f4440bf5baec9d16939e8b9c508a5fc9d841fc7b

Reply:
Wait a second, no I didn't

Reply:
Okay now it's fixed https://github.com/EngineHub/WorldEditDocs/pull/13/commits/8c59671233206d8c1a99f64ceffd966e662518b1